core/src/main/java/com/linecorp/armeria/client/ClientRequestContextWrapper.java

```diff
@@ -140,32 +140,12 @@ public void cancel(Throwable cause) {
         delegate().cancel(cause);
     }
 
-    @Override
-    public void cancel() {
-        delegate().cancel();
-    }
-
-    @Override
-    public void timeoutNow() {
-        delegate().timeoutNow();
-    }
-
     @Nullable
     @Override
     public Throwable cancellationCause() {
         return delegate().cancellationCause();
     }
 
-    @Override
-    public boolean isCancelled() {
-        return delegate().isCancelled();
-    }
-
-    @Override
-    public boolean isTimedOut() {
-        return delegate().isTimedOut();
-    }
-
```

Comment:
We still need these methods because the delegate's methods might have different behavior than the default methods.